docs/client/config.json

```diff
@@ -62,6 +62,25 @@
         }
       }
     },
+    {
+      "url": "./tmp-swagger-gen/ibc/core/channel/v2/query.swagger.json",
+      "operationIds": {
+        "rename": {
+          "Params": "ChannelV2Params",
+          "Channel": "ChannelV2",
+          "ChannelClientState": "ChannelClientStateV2",
```

Comment:
hm, dunno if it would be better to rename these for v1?

Reply:
Is there collisions here or something?

Reply:
think so, think this is why each one renames Params.